state.json

```
@@ -30,7 +30,8 @@
                     "https://are-we-triaged-yet.herokuapp.com/?version=68&report=untriaged comment=\"Nightly Triage\"",
                     "gslide id=1Nh4GKmcT6ve4VcblPLvof29zIANgD7KPQtI7aDid0Bk comment=\"nice tweets\"",
                     "https://bgrins.github.io/xbl-analysis/graph/#burndown zoom=180 comment=\"XBL Burndown\"",
-                    "https://i.imgur.com/w4epniR.png comment=\"xMatters status\" before=2019-05-16T00:00:00Z"
+                    "https://i.imgur.com/w4epniR.png comment=\"xMatters status\" before=2019-05-16T00:00:00Z",
+                    "https://i.imgur.com/ANsZWzf.jpg comment=\"Global Accessibility Awareness Day\",
```

Trailing commas can't be used in JSON, unfortunately.

```suggestion
                    "https://i.imgur.com/ANsZWzf.jpg comment=\"Global Accessibility Awareness Day\"
```